Cargo.lock

```diff
@@ -1,22 +1,3 @@
-[root]
```

Comment:
I am not sure why Cargo moved this. :-(

Reply:
I don't worry too much about Cargo changes.